TheWarWithin/RogueAssassination.lua

```diff
@@ -2173,6 +2174,14 @@ spec:RegisterAbilities( {
         aura = "garrote",
         cycle = "garrote",
 
+        usable = function ()
```

Comment:
Again, I get concerned about `usable` functions since they're a hard binary and not time sensitive.

I think we can combine the two `if`s here:

```Lua
if not debuff.garrote.refreshable and settings.max_garrote_spread > 0 and buff.indiscriminate_carnage_any.up then
    return ( active_dot.garrote < settings.max_garrote_spread ), strformat( "active Garrote DoTs higher than or equal to the user specified maximum of (%d)", settings.max_garrote_spread )
end
return true
```

Reply:
Also, these explanations are generally kept very brief based on where they fit in snapshots, if you want to trim it back to something like:

`active garrotes [%d] >= max garrote setting [%d]`

Reply:
I mentioned it in the Discord, but I suggest it again here for easier tracking.

Instead of an exact number of active Garrotes to target, just turn the slider into a 0-100 percent defaulting to 50% for the active enemies with Garrote on them.

Reply:
I can see both approaches being valuable; I know the Keg Smash use-case was something I felt out because there was *no* check at that time.  And Keg Smash applies its debuff to multiple targets in one cast, while Garrote is going to apply to 2 or 3 at a time?

Maybe that's the distinguishing factor?  Number of GCDs that would be spent on Garrote based on active_dot.garrote vs. true_active_enemies vs. Indiscriminate Carnage or other effects(?) that impact # of Garrotes applied on use?